### ISSUE TITLE
TC: New Order/WF/Multi-Part/ ~file uploader~ file name error

### ISSUE THREAD

**Issue by leobardoramon** [codetodesign/NYUAMS#718]
- @barryross , can you please look into this issue ?

 - Is this issue related to the open issue , #699 , Multiple parts for existing orders ?

Issue : File Naming Error

Related To : #neworderwizard #fileuploader #wideformatservice #invoicepdf # 

When using the **New Order Wizard** to create multiple parts for any type of **Wide Format Order** , the name from the first file gets carried over to any following files that have been uploaded .

I think that is may be related to the #filedescription field or #partdescription field , since the error does not appear in Projects or Orders , but appears in Queue and on the Invoice PDF

- @buck0land , does this issue appear in Receipt PDF ?

**Comment by leobardoramon** [comment]
@barryross , we will send you the new IP addresses for the two DEV servers sometime today for vetting .  Please let us know your availability for this week / next week .

**Comment by leobardoramon** [comment]
![screen shot 2017-11-27 at 10 17 33 am](https://user-images.githubusercontent.com/2165796/33276104-87b05d3c-d362-11e7-8928-157955cd1a31.png)
![screen shot 2017-11-27 at 10 18 00 am](https://user-images.githubusercontent.com/2165796/33276105-87be3bfa-d362-11e7-86ad-690520583869.png)
![screen shot 2017-11-27 at 10 19 23 am](https://user-images.githubusercontent.com/2165796/33276106-87c69106-d362-11e7-9cd3-46e5d22d2966.png)

**Comment by barryross** [comment]
Yes, this sounds like it may be a bug from the work on multiple parts for new orders, or perhaps something to be tackled in the multiple parts for existing orders.   My availability is a bit tight this week, but can definitely put it on my agenda for early next week if that works for you guys?
